docs/tutorials/typescript/03-policy-settings.md

```diff
@@ -0,0 +1,103 @@
+---
+sidebar_label: Defining policy settings
+sidebar_position: 030
+title: Defining policy settings
+description: Defining policy settings for a Kubewarden policy written in TypeScript.
+keywords: [kubewarden, kubernetes, defining policy settings, TypeScript]
+doc-type: [tutorial]
+doc-topic: [kubewarden, writing-policies, typescript, defining-policy-settings]
+doc-persona: [kubewarden-policy-developer]
+---
+
+<head>
+  <link rel="canonical" href="https://docs.kubewarden.io/tutorials/writing-policies/typescript/policy-settings"/>
+</head>
+
```

Comment:
I think we should add a big fat warning tooltip that reminds developers they must not print logging information to the STDOUT, otherwise the policy will be broken.

Explain they have to either write to the STDERR or use the logging facility from our SDK (do we have it?)